src/logger/logger.ts

```diff
@@ -4,6 +4,13 @@ import * as uuid from 'uuid';
 import stringify from 'fast-safe-stringify';
 import isError from 'is-error';
 
+export const redactSecret = (data: any): any => {
+  return data.replace(
```

Comment:
If `data` is `any`, can it happen that some types won't have the `.replace` method?

Reply:
Haha, asked the same https://github.com/Cimpress-MCP/lambda-essentials-ts/pull/49#discussion_r1050765272 :)

Reply:
hmm in JS it works even if the data is an object, but not in TS
Ok gonna write some unit tests...